tests/test_controller.py

```diff
@@ -779,9 +811,10 @@ def test_process_urn_isbn(self):
         self.assert_one_message(
             isbn.urn, HTTP_CREATED, self.controller.IDENTIFIER_REGISTERED
         )
-        [record] = isbn.coverage_records
-        eq_(record.exception, self.controller.NO_WORK_DONE_EXCEPTION)
-        eq_(record.status, CoverageRecord.TRANSIENT_FAILURE)
+        assert isbn.coverage_records
+        for cr in isbn.coverage_records:
+            eq_(cr.exception, self.controller.registrar.NO_WORK_DONE_EXCEPTION)
+            eq_(cr.status, CoverageRecord.TRANSIENT_FAILURE)
```

Comment:
Is it worth testing which specific tasks were created as CoverageRecords?

Reply:
I was hoping I could leave that in the tests for TestIdentifierResolutionRegistrar and just make sure it's doing the bare minimum job here.